gooddata-sdk/gooddata_sdk/catalog/workspace/declarative_model/workspace/analytics_model/analytics_model.py

```diff
@@ -0,0 +1,144 @@
+# (C) 2022 GoodData Corporation
+from __future__ import annotations
+
+from typing import Any, Type, TypeVar
+
+from gooddata_metadata_client.model.declarative_analytical_dashboard import DeclarativeAnalyticalDashboard
+from gooddata_metadata_client.model.declarative_analytics_layer import DeclarativeAnalyticsLayer
+from gooddata_metadata_client.model.declarative_dashboard_plugin import DeclarativeDashboardPlugin
+from gooddata_metadata_client.model.declarative_filter_context import DeclarativeFilterContext
+from gooddata_metadata_client.model.declarative_metric import DeclarativeMetric
+from gooddata_metadata_client.model.declarative_visualization_object import DeclarativeVisualizationObject
+
+T = TypeVar("T", bound="CatalogAnalyticsBase")
+
+
+class CatalogDeclarativeAnalyticsLayer:
+    def __init__(
+        self,
+        analytical_dashboards: list[CatalogDeclarativeAnalyticalDashboard] = None,
+        dashboard_plugins: list[CatalogDeclarativeDashboardPlugin] = None,
+        filter_contexts: list[CatalogDeclarativeFilterContext] = None,
+        metrics: list[CatalogDeclarativeMetric] = None,
+        visualization_objects: list[CatalogDeclarativeVisualizationObject] = None,
+    ):
+        self.analytical_dashboards = analytical_dashboards if analytical_dashboards else []
```

Comment:
can be simplified to:
self.analytical_dashboards = analytical_dashboards or []

Reply:
Good point! Thanks.